release_bot/github.py

```diff
@@ -518,11 +526,15 @@ def put_labels_on_issue(self, number, labels):
         :param labels: list of str
         :return: True on success, False on fail
         """
+        if self.conf.dry_run:
+            self.logger.info("I would add labels to issue #%s", number)
+            return "SKIPPED"
```

Comment:
Similar here: the expectation here is to return True/False; it's true that `True is bool("SKIPPED")`, but let's be consistent please; just return False